var/spack/repos/builtin/packages/netcdf-fortran/package.py

```diff
@@ -0,0 +1,18 @@
+from spack import *
+
+class NetcdfFortran(Package):
+    """Fortran interface for NetCDF4"""
+
+    homepage = "http://www.unidata.ucar.edu/downloads/netcdf/netcdf-cxx/index.jsp"
+    url      = "http://www.unidata.ucar.edu/downloads/netcdf/ftp/netcdf-fortran-4.4.3.tar.gz"
+
+    version('4.4.3', 'bfd4ae23a34635b273d3eb0d91cbde9e')
+
+    variant('mpi', default=True, description='Enables MPI parallelism')
```

Comment:
At one point, I had an mpi variant; I think because I didn't really
understand how Spack works.  I removed it, but forgot to remove the
variant() lines from the packages.  If others think that's the right thing
to do, I'll go ahead and remove them too.

On Sat, Mar 5, 2016 at 6:40 PM, Adam J. Stewart <notifications@github.com>
wrote:

> In var/spack/repos/builtin/packages/netcdf-fortran/package.py:
>
> > @@ -0,0 +1,18 @@
> > +from spack import *
> > +
> > +class NetcdfFortran(Package):
> > +    """Fortran interface for NetCDF4"""
> > +
> > +    homepage = "http://www.unidata.ucar.edu/downloads/netcdf/netcdf-cxx/index.jsp"
> > +    url      = "http://www.unidata.ucar.edu/downloads/netcdf/ftp/netcdf-fortran-4.4.3.tar.gz"
> > +
> > +    version('4.4.3', 'bfd4ae23a34635b273d3eb0d91cbde9e')
> > +
> > +    variant('mpi', default=True, description='Enables MPI parallelism')
>
> Using:
>
> depends_on('netcdf-fortran')
> depends_on('netcdf+mpi')
>
> wouldn't accidentally build netcdf+mpi *and* netcdf~mpi would it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LLNL/spack/pull/491/files#r55130033>.
>